spec/system/performance_tracking/viewing_spec.rb

```diff
@@ -51,26 +84,22 @@
       end
     end
 
-    context 'when logged in on staging' do
+    context 'when logged in as a user manager and supervisor, and service access is enabled' do
+      include_context 'when logged in user is admin'
+
+      let(:current_user_role) { UserRole::SUPERVISOR }
+
       before do
         allow(FeatureFlags).to receive(:allow_user_managers_service_access) {
           instance_double(FeatureFlags::EnabledFeature, enabled?: true)
         }
-        visit performance_tracking_index_path
+        visit '/'
+        click_link 'Performance tracking'
       end
 
-      it 'redirects to "Page not" found for caseworkers' do
+      it 'can access "Performance tracking" page as a user manager' do
         heading_text = page.first('.govuk-heading-xl').text
-        expect(heading_text).to eq('Page not found')
-      end
-
-      context 'when logged in as a user manager' do
-        let(:current_user_can_manage_others) { true }
-
-        it 'can access "Performance tracking" page as a user manager' do
-          heading_text = page.first('.govuk-heading-xl').text
-          expect(heading_text).to eq('Performance tracking')
-        end
+        expect(heading_text).to eq('Performance tracking')
       end
```

Comment:
It seems like there is a whole bunch of tests dedicated to testing the same behaviours with/without the feature flag enabled so maybe laying out the tests with a describe block for 'with feature flag' and 'without feature flag' and then a shared example that runs in both with different expected results will make it a bit clearer?

Reply:
I've tidied things up a bit, but left it as When can access vs When cannot access. I'll DM you